packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as AtomicBreadboxStories from './atomic-breadbox.new.stories';
+import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';
+
+<Meta of={AtomicBreadboxStories} />
+
+<AtomicDocTemplate
+  stories={AtomicBreadboxStories}
+  githubPath="search/atomic-breadbox/atomic-breadbox.ts"
+  tagName="atomic-breadbox"
+  className="AtomicBreadbox"
+>
+
+Include breadcrumbs in a search page when you want users to navigate the content using facets.
+For eaxmple, on an ecommerce website breadcrumbs help the users keep track of the selected filters and being able to confirm and adjust filtering encourages browsing.
```

Comment:
I'd remove this or replace it with a non-commerce example (since this component is in the "search" use case, no commerce).

Also, I'd add:

For the sake of consistency with other mdx, I'd add:

This component is typically placed within the "status" section of the layout.

```html
<atomic-search-interface>
  ...
  <atomic-search-layout>
    ...
    <atomic-layout-section section="main">
      ...
      <atomic-layout-section section="status">

        <atomic-breadbox></atomic-breadbox>
        
      </atomic-layout-section>
      ...
    </atomic-layout-section>
  </atomic-search-layout>
</atomic-search-interface>
```

for the sake of consistency with other mdx.